blocks/footer/footer.css

```diff
@@ -1,14 +1,312 @@
+}
+
+.footer.block .footer-links .links-column {
+  flex: 0 0 auto;
+  width: 50%;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links .links-column {
+    flex: 0 0 auto;
+    width: 16.6667%;
+    padding-left: 1rem;
+  }
+}
+
+.footer.block .footer-links .links-column > div {
+  margin: 0 0 1.25rem
+}
+
+@media (width < 37.5rem) {
+  .footer.block .footer-top-note p {
+    padding-left: 0.9375rem;
+    padding-right: 0.9375rem;
+  }
+}
+
+.footer.block .footer-top .row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -0.9rem;
+  margin-left: -0.9rem;
+}
+
+.footer.block .footer-links .row {
+  --bs-gutter-x: 1.5rem;
+  --bs-gutter-y: 0;
+
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(-1* var(--bs-gutter-y));
+  margin-right: calc(-.5* var(--bs-gutter-x));
+  margin-left: calc(-.5* var(--bs-gutter-x));
+}
+
+.footer.block .footer-links .row>* {
+  padding-right: calc(var(--bs-gutter-x)* .5);
+  padding-left: calc(var(--bs-gutter-x)* .5);
+  margin-top: var(--bs-gutter-y);
+}
+
+@media (width >= 56.25rem) {
+  .footer.block .section > .section-container {
+    max-width: 81.25rem;
+    margin: auto;
+  }
+}
+
+.footer.block .footer-disclaimer {
+  background: var(--link-color-dark-gray);
+  padding-top: 1.875rem;
+}
+
+.footer.block .footer-disclaimer a:hover {
+  color: var(--link-color-blue);
+}
+
+.footer.block .footer-disclaimer .panel-header {
+  position: relative;
+  padding: 0.625rem 0;
+  padding-right: 2.5rem;
+  z-index: 4;
+  cursor: pointer;
+}
+
+.footer.block .footer-disclaimer .panel-header strong {
+  font-size: 0.9375rem;
+  padding-right: 0.9375rem;
+  font-weight: 500;
+  background-color: var(--link-color-dark-gray);
+  z-index: 2;
+  position: relative;
+  color: var(--color-off-white);
+}
+
+.footer.block .footer-disclaimer .panel-header::before {
+  content: "";
+  position: absolute;
+  height: 0.0625rem;
+  width: 95%;
+  border-bottom: 0.0625rem dashed var(--horizontal-footer-dash-gray);
+  top: 50%;
+  right: 3.125rem;
+  left: 0;
+  transform: translateY(-50%);
+  z-index: 1;
 }
+
+.footer.block .footer-disclaimer .panel-header .toggle-icon {
+  background: var(--link-color-dark-gray);
+  font-size: var(--body-font-size-l);
+  font-style: normal;
+  position: absolute;
+  top: 0.375rem;
+  right: 0.875rem;
+  width: 1.75rem;
+  height: 1.75rem;
+  text-align: center;
+  color: var(--color-off-white);
+  border: 0.0625rem solid var(--color-off-white);
+  border-radius: 50%;
+  z-index: 10;
+}
+
+.footer.block .footer-disclaimer .panel-body {
+  padding: 0.625rem 0;
+}
+
+.footer.block .footer-disclaimer .panel-body span {
+  color: var(--color-footer-gray);
+  font-size: var(--body-font-size-s);
+}
+
+.footer.block .footer-disclaimer .section-container .default-content-wrapper > div {
+  overflow: hidden;
+  max-height: 200rem;
+  opacity: 1;
+  transition: max-height 0.4s ease, opacity 0.4s ease;
+}
+
+.footer.block .footer-disclaimer .panel-body.slide-up {
+  max-height: 0 !important;
```

Comment:
Please avoid `!important` qualifier in styles.

Reply:
yes it is working without `!important`, earlier css was not organised that's I have added but now not needed.